src/sentry/api/endpoints/organization_details.py

```diff
@@ -116,6 +124,38 @@ def update_organization_scenario(runner):
         )
 
 
+class TrustedRelaySerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        return convert_dict_key_case(instance, snake_to_camel_case)
+
+    def to_internal_value(self, data):
+
+        try:
+            key_name = data.get(u"name")
+            public_key = data.get(u"publicKey", "")
+            description = data.get(u"description")
+        except AttributeError:
+            raise serializers.ValidationError("invalid-message")
+
+        if key_name is None:
+            raise serializers.ValidationError("missing-name")
```

Comment:
Could you please change all `ValidationError`s into human readable error messages? Those will be directly rendered in the UI.

Reply:
ok, done

Reply:
Done